account-kit/react/src/hooks/useAuthenticate.ts

```diff
@@ -67,6 +77,7 @@ export function useAuthenticate(
   );
 
   return {
+    preparePopupOauth,
```

Comment:
this is here in case a developer wants to manually prepare the popup? doesn't the config handle this for you now?